docs/provider-list.mdx

```diff
@@ -10,16 +10,19 @@ import TestProvider from '@site/src/components/TestProvider';
 This list contains providers that have been tested with MCP Auth.
 
 | Provider                             | Type           | OAuth 2.1 | Metadata URL | Dynamic Client Registration | Resource Indicator |
```

Comment:
@xiaoyijun Wasn't quite sure what Resource Indicator meant here myself. Perhaps adding some docs would be useful as well?

Reply:
I have added the Resource Indicator rfc link to the table.

Reply:
@xiaoyijun Thanks! Can you explain why this is important to call out? I think linking the relevant parts of the MCP spec would be useful in this document as well.